### DISCUSSION TITLE
[Script request] Woodpecker CI

### DISCUSSION THREAD

**Discussion by adaszko** [community-scripts/ProxmoxVE#728] [Request script]
### Application Name

Woodpecker CI

### Website

https://woodpecker-ci.org/

### Description

I'm currently hosting my repos on Forgejo (thank you for the script BTW!) and it's working great but I lack running tests on every push.  There's [Forgejo Actions](https://forgejo.org/docs/latest/user/actions/) but it looks pretty new and setting it up requires additional manual setup not covered by the script to host [forgejo runner](https://code.forgejo.org/forgejo/runner) which is also alpha-quality software.  Same applies for Gitea (Actions).

OTOH, Woodpecker looks more mature and has [an ecosystem of plugins available](https://woodpecker-ci.org/plugins).  I'm also hoping I could use it with external code forges like GitHub.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by wommy** [comment]
i second thing, ive been running actions in a separate docker lxc and it was an absolute bear to setup and it only works half of the time
